tests/smoke/go.mod

```diff
@@ -3,52 +3,67 @@ module github.com/kyma-project/lifecycle-manager/tests/smoke
 go 1.19
 
 require (
-	github.com/stretchr/testify v1.8.0
-	go.uber.org/zap v1.23.0
 	gopkg.in/yaml.v3 v3.0.1
 	k8s.io/api v0.25.2
 	k8s.io/apimachinery v0.25.2
 	sigs.k8s.io/e2e-framework v0.0.7
 )
 
+require github.com/kyma-project/lifecycle-manager/operator v0.0.0-20220930074956-6267039e7d63
```

Comment:
this require is just play as a placeholder, and we don't need to maintain the version update, right?

Reply:
if it's true, maybe add a comment for this require